File: src/hacker_earth/basics_operators/Birthday.java
package hacker_earth.basics_operators;

import java.util.Scanner;

public class Birthday {

    public static void main(String[] arg) {
        Scanner br = new Scanner(System.in);
        int tCases = br.nextInt();
        while (tCases > 0) {
            int friends = br.nextInt();
            int chocolates = br.nextInt();
            if ((chocolates % friends) == 0) {
                System.out.println("Yes");
            } else {
                System.out.println("No");
            }
            tCases--;
        }
    }
}

File: src/basics_operators/BirthDay.kt
package basics_operators

import java.util.*


fun main(args: Array<String>) {

    val sc = Scanner(System.`in`)

    var tCases = sc.nextInt()
    while (tCases > 0) {
        val friends = sc.nextInt()
        val chocolates = sc.nextInt()
        if (chocolates % friends == 0) {
            println("Yes")
        } else {
            println("No")
        }
        tCases--
    }
}